tests/integration/helpers/0_common_instance_config.yaml

```diff
@@ -0,0 +1,6 @@
+timezone: Europe/Moscow
```

Comment:
I created additional files for new YAML config tests and it seems that <users_config> tag somehow forces ClickHouse to use standart users.xml file instead of users.yaml for tests. Are there any ways to fix it without adding new 0_common_instance_config files for yaml configs?